internal/helper/yawollet.go

```diff
@@ -305,6 +307,16 @@ func createEnvoyTCPListener(
 		idleTimeout = durationpb.New(lb.Spec.Options.TCPIdleTimeout.Duration)
 	}
 
+	connectionLimit, err := anypb.New(&envoyconnection.ConnectionLimit{
+		StatPrefix: "envoyconnectionlimit",
+		// this is limited by the number of available ports on the machine
+		MaxConnections: wrapperspb.UInt64(60000),
+	})
+
+	if err != nil {
+		panic(err)
```

Comment:
Maybe just return with error and post a event to the lbm/lb object. A panic inside the yawollet is hard to find and see.